### PR TITLE
Persistent storage

### PR DESCRIPTION
**Pull request by StewartJingga**

### CODE REVIEW

File: recipes/store.go [hunk 1]
```diff
@@ -6,46 +6,3 @@ type Store interface {
 	GetByName(string) (domain.Recipe, error)
```
Comment (ravisuhag) [inline recipes/store.go:6]: We can move this package to root and call it storage. As it used by recipe but not directly dependent.

Comment (StewartJingga) [reply recipes/store.go]: done! also changed a few things to accommodate it.

Comment (ravisuhag) [reply recipes/store.go]: Does it make sense to name it storage? as only one backend will be enabled at one time.

Comment (StewartJingga) [reply recipes/store.go]: Tbh I don't have any preferences between store and storage as long as it is immediately understandable and not ambiguous. Also it should be easy change, so are you talking only about the package naming or the whole thing (e.g. `RecipeStorage`)?

Comment (ravisuhag) [reply recipes/store.go]: Ok. Let's keep it store then.

**Pull request merged**